render.yml

```
@@ -0,0 +1,24 @@
+databases:
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)